### DISCUSSION TITLE
X and Y coordinate in the screen or Window

### DISCUSSION THREAD

**Discussion by vanheartnet** [Spawnova/ShinsImageScanClass#8] [Q&A]
is there an option for X and Y coordinates in the screen or window to search to using scanImage?
for example, Im using scanimage to search for a specific image but i need it to scan only in specific coordinates not the whole window or screen. ill use it to draw multiple shapes or lines at the same time for each image that are found in every Coordinates given. (X1,Y1,X2,Y2)

note: Image to search is moving in a random direction

**Comment by Spawnova** [comment]
Yeah, 
`ImageRegion(image,x1,y1,w,h,variance=0,ByRef returnX=0,ByRef returnY=0,centerResults=1,scanDir:=0)` 
ImageRegion() scans for in image in a specified region, just keep in mind that by default the class uses client space which does not include the title bar, so make sure your coordinates are client space, or specify 0 as second param when creating the class to use window space 
`scan := new ShinsImageScanClass("title",0) ;uses window space`

**Comment by vanheartnet** [comment]
Thank You! Hoping for more class and tutorial videos....
